runtime/near-wallet-contract/implementation/wallet-contract/src/internal.rs

```diff
@@ -11,9 +11,12 @@ use crate::{
 use aurora_engine_transactions::{EthTransactionKind, NormalizedEthTransaction};
 use base64::Engine;
 use ethabi::{ethereum_types::U256, Address};
-use near_sdk::{AccountId, NearToken};
+use near_sdk::{env, AccountId, NearToken};
 
-// TODO(eth-implicit): Decide on chain id.
+/// The chain ID is pulled from a file to allow this contract to be easily
+/// compiled with the appropriate value for the network it will be deployed on.
+/// The chain ID for Near mainnet is [397](https://chainlist.org/chain/398)
```

Comment:
https://chainlist.org/chain/397

Reply:
Fixed in https://github.com/near/nearcore/pull/11765/commits/b1fff0c308db155c10171289c8c8c5512643d97e